src/Messages.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { connect } from 'react-redux'
+import Message from './Message';
+
+const Messages = (props) => {
+  const { messages } = props;
+  return (
+    messages.map((message, index) => (
```

Comment:
I feel like React will show a warning if you have a return that returns more than one top-level component. I didn't get to reproduce this, but I feel like this `map` needs to be wrapped in a `Fragment` or a `<>`